### PR TITLE
feat: create 2021-10-05-è-cominciata-Hacktoberfest-2021.md

### PR DESCRIPTION
**Pull request by Francescopog**
## Description

This PR tackles:

* ...
* ...
* ...

In particular, the ...

## Checklist

* [ ] I followed the indications in [CONTRIBUTING.md](https://github.com/italia/developers.italia.it/blob/master/CONTRIBUTING.md)
* [ ] The documentation related to the proposed change has been updated accordingly (also comments in code).
* [ ] Have you written new tests for your core changes, as applicable?
* [ ] Have you successfully ran tests with your changes locally?
* [ ] Ready for review! :rocket:

## Fixes

Fixes #

### CODE REVIEW

**Comment by developers-italia-droid** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>There is a problem with the commit message
> Create 2021-10-05-è-cominciata-Hacktoberfest-2021.md
- subject may not be empty
- type may not be empty</td>
    </tr>
  </tbody>
</table>




<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 7099dbd12507a58f985217e8ef038f74f5a8aa27
</p>

File: _posts/it/2021-10-05-è-cominciata-Hacktoberfest-2021.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+layout: post
```
Comment (bfabio) [inline _posts/it/2021-10-05-è-cominciata-Hacktoberfest-2021.md:1]:
```suggestion
---
layout: post
```

Le divisioni coi trattini vanno sempre messi perché fanno parte della sintassi dell'intestazione iniziale

File: _posts/it/2021-10-05-è-cominciata-Hacktoberfest-2021.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+title: "É cominciata l'Hacktoberfest 2021"
+subtitle: Partecipa all’evento internazionale per favorire lo sviluppo di software open source. Developers Italia ci sarà
+date: 2021-10-05 10:00:00 +0200
+tags: developers opensource Hacktoberfest
+categories: news
+author: Andrea Stagi
+image: /assets/icons/logo-it.png
+locale: it
```
Comment (bfabio) [inline _posts/it/2021-10-05-è-cominciata-Hacktoberfest-2021.md:9]:
```suggestion
locale: it
---
```

File: _posts/it/2021-10-05-è-cominciata-Hacktoberfest-2021.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
+layout: post
+title: "É cominciata l'Hacktoberfest 2021"
```
Comment (bfabio) [inline _posts/it/2021-10-05-è-cominciata-Hacktoberfest-2021.md:2]:
```suggestion
title: "È cominciata l'Hacktoberfest 2021"
```

File: _posts/it/2021-10-05-è-cominciata-Hacktoberfest-2021.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+subtitle: Partecipa all’evento internazionale per favorire lo sviluppo di software open source. Developers Italia ci sarà
+date: 2021-10-05 10:00:00 +0200
+tags: developers opensource Hacktoberfest
```
Comment (bfabio) [inline _posts/it/2021-10-05-è-cominciata-Hacktoberfest-2021.md:5]:
```suggestion
tags: developers opensource hacktoberfest
```
I tag sono sempre minuscoli

**Review by bfabio — CHANGES_REQUESTED** [review]

File: _posts/it/2021-10-05-è-cominciata-Hacktoberfest-2021.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
+---
+layout: post
+title: "Comincia l'hacktoberfest 2021"
```
Comment (bfabio) [inline _posts/it/2021-10-05-è-cominciata-Hacktoberfest-2021.md:3]:
```suggestion
title: "Comincia l'Hacktoberfest 2021"
```

File: _posts/it/2021-10-05-è-cominciata-Hacktoberfest-2021.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+
+Nel corso del mese sarà anche possibile contribuire ai progetti software ospitati all’interno delle organizzazioni [Github/italia](https://github.com/italia){:target="_blank"} e [/teamdigitale](https://github.com/teamdigitale){:target="_blank"}. Questo permetterà di supportare lo sviluppo dei servizi pubblici digitali e, di conseguenza, di **migliorare il rapporto tra cittadini e Pubblica amministrazione**.
```
Comment (bfabio) [inline _posts/it/2021-10-05-è-cominciata-Hacktoberfest-2021.md:17]:
```suggestion
Nel corso del mese sarà anche possibile contribuire ai progetti software ospitati all’interno delle organizzazioni GitHub [/italia](https://github.com/italia){:target="_blank"} e [/teamdigitale](https://github.com/teamdigitale){:target="_blank"}. Questo permetterà di supportare lo sviluppo dei servizi pubblici digitali e, di conseguenza, di **migliorare il rapporto tra cittadini e Pubblica amministrazione**.
```

**Review by bfabio — APPROVED** [review]

**Pull request merged**